migrations/2_deploy_contracts.js

```diff
@@ -39,7 +39,7 @@ module.exports = function(deployer) {
   deployer
     .then(() => {
       console.log([name, deposit,limitOfParticipants, coolingPeriod, encryption].join(','));
-      return deployer.deploy(Conference, name, deposit,limitOfParticipants, coolingPeriod, encryption);
+      return deployer.deploy(Conference, name, deposit,limitOfParticipants, coolingPeriod, encryption, owner);
```

Comment:
Probably no need to pass `owner` here?

Reply:
Or pass `0x` to use the default value (= `msg.sender`). By doing so we can get rid of `owner` variable altogether as we are also getting rid of ENS contracts from migration